modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -63,7 +63,7 @@ import { supportedCrossChainRecoveries } from './config';
 
 const { getExternalChainCode, isChainCode, scriptTypeForChain, outputScripts, toOutput, verifySignatureWithUnspent } =
   bitgo;
-type Unspent<TNumber extends number | bigint = number> = bitgo.Unspent<TNumber>;
+export type Unspent<TNumber extends number | bigint = number> = bitgo.Unspent<TNumber> & { valueString?: string };
```

Comment:
We should add a separate type `UnspentJSON` instead, like I suggested here: https://github.com/BitGo/BitGoJS/pull/2785#discussion_r991886762

This change will mix up internal representation and the representation used when communicating with WP